src/MusicStore/Startup.cs

```diff
@@ -16,6 +16,8 @@ namespace MusicStore
 {
     public class Startup
     {
+        public IConfiguration Configuration { get; private set; }
```

Comment:
The pattern we use nearly everywhere is:

1. Fields
2. Constructors
3. Properties
4. Events (which we almost never have these days)
5. Methods
6. Nested types

In my mind the rationale for this is that that's the sequence of things that execute in a class: fields are initialized, constructors are called, properties/events are set/wired-up, methods are executed, and the nested types are used as implementation details.